universal-application-tool-0.0.1/conf/messages.en-US

```diff
@@ -12,7 +12,7 @@ button.continue=Continue
 button.deleteFile=Delete
 button.keepFile=Continue
 button.logout=Logout
-button.nextScreen=Next
+button.nextScreen=Save And Next
```

Comment:
probably would want the 'And' to be lowercased?

Reply:
Everything is shown as upper case text in UI

Reply:
Everything is shown as upper case text in UI